djangocms_navigation/admin.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib import admin
+from treebeard.admin import TreeAdmin
+from treebeard.forms import movenodeform_factory
+
+from .models import Menu, MenuContent, MenuItem
+
+
+class MenuContentAdmin(admin.ModelAdmin):
+    exclude = ['menu', ]
+
+    def save_model(self, request, obj, form, change):
+        if not obj.id:
```

Comment:
Maybe add a comment about why we're doing this here. It's the sort of thing that could confuse somebody

Reply:
`if not change`?